### ISSUE TITLE
[BUG] Actual bug: extra "2021" folder is created in hhmmss leaf

### ISSUE THREAD

**Issue by zilexa** [3coma3/btrfs-backup#22]
The backup runs fine, but when it is finished, an unexpected folder is created in the hhmmss folder (subvolume), besides the source folder. This folder is called 2021, contains a 01 subfolder that is empty. 

This should not happen.

Screenshot: 
![Screenshot from 2021-01-24 21-44-45](https://user-images.githubusercontent.com/3430004/105643077-16740000-5e8e-11eb-91e7-1d41b15ebba5.png)

Conf file:
```
# Snapshot of user

# To be run nightly via cron

destination='/mnt/disks/disk5/Users/Shanti/.snapshots'

sources+=('Shanti')

filters['Shanti']="
- .filerun*
- .Trash-*
- Thumbs.db
- RECYCLE.BIN
- .Spotlight-V100
- .TemporaryItems
- .Trashes
- .fseventsd
- .DS_Store
"
```

**Comment by 3coma3** [comment]
Thanks for reporting, I'll try to test and reproduce this.

If someone else sees this and can confirm, it would be very helpful too!

**Comment by 3coma3** [comment]
Fixed by b8aee12
